content/en/news/releases/1.13.x/announcing-1.13.9/index.md

```diff
@@ -7,7 +7,10 @@ publishdate: 2022-10-11
 release: 1.13.9
 ---
 
-This release contains a patch for [CVE-2022-41715](https://github.com/golang/go/issues/55949) and bug fixes to improve robustness. This release note describes what is different between Istio 1.13.8 and Istio 1.13.9.
+This release contains a fix for [CVE-2022-39278](/news/security/istio-security-2022-007/#cve-2022-39278) and
+bug fixes to improve robustness. We recommend users install this release instead of Istio 1.13.8,
```

Comment:
I'm not sure we need the `We recommend...` sentence in the announcement. We did use it for the case where a build regressed a CVE fixed in the prior build, and we really didn't want someone to move to the release with the missing fix, but instead go one further release.  Looking back at an earlier announcements, we simply `This release fixes the security vulnerabilities described in our June 9th post, [ISTIO-SECURITY-2022-005](https://istio.io/latest/news/security/istio-security-2022-005). This release note describes what’s different between Istio 1.12.7 and 1.12.8.` which didn't have that sentence.